experimental/python/pyproject.toml

```diff
@@ -19,7 +19,7 @@ dev-dependencies = [
   "pyright==1.1.380",
   "pytest-cov==5.0.0",
   "pytest==8.3.3",
-  "ruff==0.6.8",
+  "ruff==0.9.1",
```

Comment:
It's necessary because otherwise ruff in GH actions doesn't agree with ruff in experimental/python

Reply:
(ideally this is a separate PR)

Reply:
:+1:, fixed in https://github.com/databricks/cli/pull/2566